README.md

```diff
@@ -4,4 +4,4 @@ Collection of custom GitHub Actions.
 | Action | Summary |
 | - | - |
 | [prepare-img-tag](prepare-img-tag) | Prepare a tag for Docker images and decide if they should be published. |
-| [bandit-python](bandit-github)| Posts [bandit](https://github.com/PyCQA/bandit) security issues as a comment in a pull request. |
+| [bandit-github](bandit-github)| Posts [bandit](https://github.com/PyCQA/bandit) code security issues as a comment in a pull request. |
```

Comment:
I personally prefer the old name :grinning: I mean, it's obvious that GitHub Actions is related to GitHub, but the term "python" is a useful information.

Reply:
Ketzer

Reply:
![image](https://user-images.githubusercontent.com/16133277/167622192-389c3518-b9a7-45e7-804b-35392181fe44.png)